docs/index.md

```diff
@@ -152,6 +152,17 @@ bundle: {
 }
 ```
 
+For deterministic operations, it is possible to pin a module version by its OCI digest:
```

Comment:
Maybe include an example here of version and digest?

Reply:
This level of detail would require a doc dedicated to bundles, where we explain the behaviour of combining version with digest. Going to make an issue for documenting bundles.